app/package.json

```diff
@@ -26,7 +26,8 @@
     "cy:run": "cypress run",
     "cy:test": "start-server-and-test start http://localhost:3000/en/auth/login cy:run",
     "prettier": "npx prettier . --write",
-    "email": "email dev --dir src/lib/emails"
+    "email": "email dev --dir src/lib/emails",
+    "create-admin": "tsx scripts/create-admin.ts"
```

Comment:
Does this work? I don't think we have tsx in our Docker image. I've run into a similar problem with the catalog sync.

Reply:
You're right. tsx is currently added as a dev dependency. Not sure if we have to transpile it at build time with a custom task or if adding tsx to the main dependencies is fine?

Reply:
Added tsx to normal dependencies for now